service/worker/scheduler/spec.go

```diff
@@ -255,11 +256,15 @@ func validateInterval(i *schedpb.IntervalSpec) error {
 	if i == nil {
 		return errors.New("interval is nil")
 	}
+	if err := timestamp.ValidateProtoDuration(i.Interval); err != nil {
+		return errors.New(fmt.Sprintf("invalid interval: %s", err.Error()))
```

Comment:
```suggestion
		return fmt.Errorf("invalid interval: %w", err)
```
but it still will complain...